bitnami/kafka/Chart.yaml

```
@@ -42,4 +42,4 @@ maintainers:
 name: kafka
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/kafka
-version: 26.2.0
+version: 26.2.1
```

Though this change could be considered a fix in some context, I would say it is more of a feature addition. As such, the version bump should be to a new minor instead:

```suggestion
version: 26.3.0
```